storybook/preview.tsx

```diff
@@ -1,59 +1,23 @@
 /* eslint import/no-extraneous-dependencies: 0 */
 import "./tailwind.scss"
 import React from "react"
-import { addParameters } from "@storybook/react"
-import { QueryClientProvider, QueryClient } from "@tanstack/react-query"
+import { Preview } from "@storybook/react"
 import { defaultTheme, ThemeContext } from "@kaizen/design-tokens"
 import { backgrounds } from "./backgrounds"
-import { CATEGORIES, SORT_ORDER } from "./constants"
 import "highlight.js/styles/a11y-light.css"
 
-const queryClient = new QueryClient()
-
-const withQueryProvider = (Story): JSX.Element => (
-  <QueryClientProvider client={queryClient}>{Story()}</QueryClientProvider>
-)
 // Polyfill for :focus-visible pseudo-selector
 // See: https://github.com/WICG/focus-visible
-require("focus-visible")
+import "focus-visible"
 
 // Standard base stylesheet used across Culture Amp products
 // See: https://github.com/necolas/normalize.css/
-require("normalize.css")
-
-require("@kaizen/component-library/styles/fonts.scss")
-require("./global.scss")
+import "normalize.css"
 
-addParameters({
-  backgrounds: {
-    default: "White",
-    values: backgrounds,
-  },
-  options: {
-    storySort: {
-      method: "alphabetical",
-      order: [
-        CATEGORIES.introduction,
-        ...SORT_ORDER.systems,
-        CATEGORIES.components,
-        CATEGORIES.helpers,
-        CATEGORIES.designTokens,
-        CATEGORIES.deprecated,
-      ],
-    },
-  },
-  docs: {
-    extractComponentDescription: (component, { notes }) => {
-      if (notes) {
-        return typeof notes === "string" ? notes : notes.markdown || notes.text
-      }
-      return null
-    },
-  },
-  chromatic: { disable: true },
-})
+import "@kaizen/component-library/styles/fonts.scss"
+import "./global.scss"
```

Comment:
`tailwind.scss` is the new `global.scss` I found that global doesn't get loaded for some reason...

Reply:
Deleted :)